packages/auth/demo/README.md

```diff
@@ -44,6 +44,7 @@ in the `config.js` file.
 
 Before deploying, you may need to build the auth package:
 ```bash
+cd auth/demo
```

Comment:
Can we also add:

You can optionally clear the cache and rebuild using `rm -rf node_modules && yarn build:deps`

Reply:
Added